service/common/src/test/java/org/apache/polaris/service/exception/IcebergExceptionMapperTest.java

```diff
@@ -16,7 +16,7 @@
  * specific language governing permissions and limitations
  * under the License.
  */
-package org.apache.polaris.service.quarkus;
+package org.apache.polaris.service.exception;
```

Comment:
nit: ideally simple renames/moves should be done in isolated PRs... it does not look like this change is related to the logging changes in other classes :)

Reply:
Agree in general. This one seemed trivial since the test is isolated and was moved at some point during the Quarkus refactor to be in a different package than IcebergExceptionMapper.

Reply:
yeah, no worries